Vagrantfile

```diff
@@ -168,10 +168,6 @@ Vagrant.configure(2) do |config|
       sudo cp /vagrant/commissaire-service/conf/systemd/commissaire-investigator.service /etc/systemd/system/commissaire-investigator.service
       sudo sed -i 's|^ExecStart=.*|ExecStart=/bin/bash -c ". /home/vagrant/commissaire_env/bin/activate \\&\\& commissaire-investigator-service --bus-uri redis://192.168.152.101:6379"|' /etc/systemd/system/commissaire-investigator.service
 
-      echo "===> Setting up commissaire-watcher service to autostart"
```

Comment:
It interferes with the E2E tests by intermittently injecting additional notifications on the bus that the tests aren't expecting.  Disabling it until I can figure out a better solution.